src/include/sof/lib/uuid.h

```diff
@@ -40,6 +40,18 @@ struct sof_uuid {
 	uint8_t  d[8];
 };
 
+/**
+ * \brief Connects UUID with component description
+ *
+ * Declaration of this structure should be done by DECLARE_SOF_UUID(),
+ * then declaration will be part of `.static_uuids` section and `SMEX` tool
+ * use it during `ldc` file creation.
+ */
+struct sof_uuid_entry {
+	struct sof_uuid id;
+	const char name[32];
```

Comment:
Do we have a UUID name size macro that we can use here ?

Reply:
Atm. we don't have, would you like to add such a macro?

Reply:
It's probably a good idea, yes.

Reply:
Done, `UUID_NAME_MAX_LEN 32` I guess it should be enough